lib/omnibus/builder.rb

```diff
@@ -217,6 +217,10 @@ def configure(*args)
       # Accept a prefix override if provided. Can be set to '' to suppress
       # this functionality.
       prefix = options.delete(:prefix) || "#{install_dir}/embedded"
+
+      if windows_target?
+        prefix.sub! "C:/", "/c/"
```

Comment:
❓ Question :
Looks good but to confirm in which way is it replacing ? since I'd expect `C:/` path to be the standard on windows and `/c/` not working

Reply:
Indeed that only work when running the build under msys or similar, definitely not in powershell... 
That should be fine since I'm pretty sure autoconf will not work anyway, but I'll double check with the windows team